kit.yml

```diff
@@ -165,6 +170,8 @@ credentials:
     tsa/host_key: ssh 2048 fixed
     tsa/worker_key: ssh 2048 fixed
 
+    atc/token_signing_key: rsa 2048 fixed
```

Comment:
Just to check my assumption, the ATC Token Signing Key is fixed because it signs long-lived tokens that are outside the scope of the deployment (the fly cli) and rotating it regularly would cause a bad end-user experience? (_why do I keep getting logged out?!_)

Reply:
That was the idea, yeah. Whether or not this should be "fixed" is questionable, because its not so bad if its a once a month cycle, but daily or weekly cycles of this could be frustrating. I chose to err on the safe side.

I'm not sure why host and worker keys are fixed though. I mean, if you have external deployments of the workers... then, sure, worker key. Host key is between atc and tsa, so... is should that really be fixed?

Reply:
I take it back. Its for the database. It would need to be fixed if its an external deployment or provided by your IaaS